plugins/aea-ledger-fetchai/aea_ledger_fetchai/_cosmos.py

```diff
@@ -1333,9 +1363,10 @@ def _try_create_faucet_claim(
 
         uid = None
         if response.status_code == 200:
-            data = response.json()
-            uid = data["uid"]
-
+            try:
+                uid = response.json()["uid"]
```

Comment:
is `KeyError` the only exception that can be raised here? What about `JSONDecodeError`: https://docs.python-requests.org/en/master/user/quickstart/#json-response-content